### PR TITLE
8028265: Move closed timezone tests to open repository

### PR DESCRIPTION
**Pull request by satoyoshiki**
/reviewer @coffeys

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [ ] Change must be properly reviewed

### Issue
 * [JDK-8028265](https://bugs.openjdk.java.net/browse/JDK-8028265): Move closed timezone tests to open repository

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/5155/head:pull/5155` \
`$ git checkout pull/5155`

Update a local copy of the PR: \
`$ git checkout pull/5155` \
`$ git pull https://git.openjdk.java.net/jdk pull/5155/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 5155`

View PR using the GUI difftool: \
`$ git pr show -t 5155`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/5155.diff">https://git.openjdk.java.net/jdk/pull/5155.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/util/TimeZone/BogusRuleLengthTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
```
Comment (AlanBateman) [inline test/jdk/java/util/TimeZone/BogusRuleLengthTest.java:4]: Please make sure to change or add GPL headers to these tests.

**Comment by coffeys** [comment]
this PR is premature. @satoyoshiki is going to close it out and re-work it.

**Pull request closed without merging**